aineko/core/config_loader.py

```diff
@@ -3,8 +3,8 @@
 """Module to load config files."""
 from typing import Optional, Union, overload
 
-from schema import Optional as optional
-from schema import Schema, SchemaError
+from schema import Optional as optional  # type: ignore
+from schema import Schema, SchemaError  # type: ignore
```

Comment:
```suggestion
from schema import Optional as optional
from schema import Schema, SchemaError
```
This has been added to the pyproject.toml mypy overrides, so this inline ignore is not needed.
https://github.com/aineko-dev/aineko/blob/d472c3dee69ae26a4bafe62e6ead4ace73c13551/pyproject.toml#L522-L524

mypy was giving you trouble because you created a mypy.ini that takes precedence over the settings in pyproject.toml